server/src/controllers/ScheduleController.js

```diff
@@ -0,0 +1,102 @@
+const ScheduleServices = require('../services/ScheduleServices');
+
+const ScheduleController = {
+  /**
+   * Get an array of events for a frosh group.
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Function} next
+   * @return {Object[]}
+   */
+  async getGroupSchedule(req, res, next) {
+    const { froshGroupId } = req.params;
+    const schedule = await ScheduleServices.getGroupSchedule(froshGroupId);
+    res.status(200).send({ groupSchedule: schedule });
```

Comment:
Please wrap this in a try catch so that if there are any errors we handle them appropriately